sorc/orog_mask_tools.fd/inland.fd/inland.F90

```diff
@@ -1,8 +1,11 @@
 !> @file
-!! This program creates the inland mask and writes it to the orography data files.
+!! Program and associated routines for creating inland mask
 !! 
 !! @author Ning Wang @date July 1, 2020
 !!
+!> This program creates the inland mask and writes it to the orography data files.
+!! @author Ning Wang
```

Comment:
Always put the @author tag last, after all the @param tags.

ALways include one blank line before the first @param. Like this:

```
!> Some brief description.
!!
!! @param a description of a
!! @author Freddy Fortran
```